Recrutify/Recrutify.Services/Validators/BaseProjectValidator.cs

```diff
@@ -109,5 +110,16 @@ private Task<bool> CheckPrimarySkillsAsync(IEnumerable<ProjectPrimarySkillDTO> p
             var ids = primarySkillDTOs.Select(x => x.Id).ToList();
             return PrimarySkillRepository.ExistsByIdsAsync(ids, cancellation);
         }
+
+        private bool LinkMustBeAUri(string link)
+        {
+            if (string.IsNullOrWhiteSpace(link))
+            {
+                return false;
+            }
+
+            Uri result;
+            return Uri.TryCreate(link, UriKind.Absolute, out result);
```

Comment:
You can write out var result, so that can remove line 121

Reply:
done